assets/section-footer.css

```diff
@@ -302,12 +301,34 @@
 }
 
 .localization-form__select {
-  box-shadow: 0 0 0 0.1rem rgba(var(--color-foreground), 0.55);
+  border-radius: var(--inputs-radius-outset);
   position: relative;
   margin-bottom: 1.5rem;
-  border: 0.1rem solid transparent;
   padding-left: 1rem;
   text-align: left;
+  min-height: calc(4rem + var(--inputs-border-width) * 2);
+  min-width: calc(7rem + var(--inputs-border-width) * 2);
+}
+
+.disclosure__button.localization-form__select {
+  padding: calc(1.5rem + var(--inputs-border-width))
+}
+
+.localization-form__select:before {
+  content: '';
+  position: absolute;
+  inset: 0;
+  border-radius: var(--inputs-radius-outset);
+  box-shadow: var(--inputs-shadow-horizontal-offset) var(--inputs-shadow-vertical-offset) var(--inputs-shadow-blur-radius) rgba(var(--color-base-text), var(--inputs-shadow-opacity));
+}
+
+.localization-form__select:after {
+  content: '';
+  position: absolute;
+  inset: var(--inputs-border-width);
+  border: 0.1rem solid transparent;
+  border-radius: var(--inputs-radius);
+  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground),var(--inputs-border-opacity)); 
```

Comment:
I would consider this to be somewhat low severity since it's specifically on focus and is just a small visual bug.  Can we create a Severity 2 or 3 issue to deal with this? @melissaperreault